tfprotov5/resource.go

```diff
@@ -609,7 +609,10 @@ type MoveResourceStateRequest struct {
 	TargetTypeName string
 
 	// SourceIdentity is the identity of the source resource.
-	SourceIdentity *ResourceIdentityData
+	SourceIdentity *RawState
```

Comment:
Probably worth clarifying now, similar to `SourceState`
```suggestion
	//
	// Only the underlying JSON field is populated.
	SourceIdentity *RawState
```